templates/generic/digest.html

```diff
@@ -0,0 +1,89 @@
+{{period}} github digest ({{#topic}}{{.}}{{/topic}}{{^topic}}{{#repos}}{{shortname}}{{^last}}, {{/last}}{{/repos}}{{/topic}})
+
+<html lang="en">
+<head>
+<meta charset="utf-8">
+<title>Weekly digest</title>
+<style>
+body { font-family: Gotham, "Helvetica Neue", Helvetica, Arial, sans-serif; }
+h2 { margin-top: 3em; }
+h3 { margin-bottom:0; margin-top: 2em; font-size: 1.4em; }
+a { color: #e6ad49; text-decoration: none; }
+li { margin-bottom: .2em; }
+.new { color: red; }
+</style>
+</head>
+
+<body>
+<h1>Date here</h1>
```

Comment:
Use {{date}} (instead of "Date here") - that way I'll be sure to remember to add that to the input to the template